deployment/smp-server-linode.sh

```diff
@@ -0,0 +1,76 @@
+#!/bin/bash
+# <UDF name="enable_store_log" label="Enable store log - enable saving SMP queues to append only log, and restoring them when the server is started." default="on" oneof="on, off" />
+# <UDF name="api_token" label="API Token - enable StackScript to create tags address and hash for SMP server domain/ip address and transport key hash from inside linode. Use "address:port#hash" as SMP server URL. Note: minimal permissions token should have are the following: Account - read/write, Linodes - read/write" default="" />
+# <UDF name="domain_address" label="Domain address" default="" />
+
+# log all stdout output to stackscript.log
+exec &> >(tee -i /var/log/stackscript.log)
+# enable debugging features
+set -xeo pipefail
+
+cd $HOME
+
+# download latest release
+curl -s https://api.github.com/repos/simplex-chat/simplexmq/releases/latest \
+| grep "browser_download_url.*20_04-x86-64*" \
+| cut -d : -f 2,3 \
+| tr -d \" \
+| wget -qi -
+
+# move smp-server executable to /opt/simplex/bin
+mv smp-server-ubuntu-20_04-x86-64 smp-server
+chmod +x smp-server
+mkdir -p /opt/simplex/bin
+mv ./smp-server /opt/simplex/bin
+
+# add /opt/simplex/bin to $PATH
+echo "export PATH=\"\$PATH:/opt/simplex/bin\"" >> .profile
+source ~/.profile
+
+# initialize SMP server
+mkdir /etc/opt/simplex
+mkdir /var/opt/simplex
+
+# -l enables store log
+if [ $ENABLE_STORE_LOG == "on" ]; then
+        smp-server init -l > simplex.conf
+else
+        smp-server init > simplex.conf
+fi
```

Comment:
maybe just provide a field to the user where he can give arbitrary options, instead of a field for env variable?
i.e. ask for "smp-server executable options", have `-l` as default

Reply:
it's just way more clear by allowing the user to go read documentation and figure out which options he wants to run with, rather than baking this logic inside the linode script via env variables

Reply:
The best I could find was 
`init_opts=()
[[ $ENABLE_STORE_LOG == "on" ]] && init_opts+=(-l)
smp-server init "${init_opts[@]}" > simplex.conf`

Reply:
![image](https://user-images.githubusercontent.com/39944650/119358633-7a2dfe80-bcb1-11eb-8010-388fcebdcb9f.png)
we can discuss doing it like this, however to me it seems that for regular user (not someone accustomed to command options) simple radiobuttons for options could be easier to understand

Reply:
one could argue that the person doing deployment would be capable of reading readme and adding command line options.. I was thinking of just a text field instead of a dropdown.

the approach with a radio button flag just doesn't scale well.. but the new implementation seems better, yep

Reply:
it's for lazy people - they don't read readme...